.vsts-dotnet-ci.yml

```diff
@@ -177,11 +144,11 @@ jobs:
 
 - template: /build/ci/job-template.yml
   parameters:
-    name: Windows_x86_NetCoreApp21
+    name: Windows_x86_NetCoreApp31
     architecture: x86
     buildScript: build.cmd
     innerLoop: true
-    vsTestConfiguration: "/Framework:.NETCoreApp,Version=v2.1"
+    vsTestConfiguration: "/Framework:.NETCoreApp,Version=v3.0"
```

Comment:
Why is this 3.0 and not 3.1?

Reply:
I copied it from [here](https://github.com/dotnet/machinelearning/blob/793872853c4af040026a7825381be567ec25da92/.vsts-dotnet-ci.yml#L119) which was there already. Should it be 3.1? If so, I will go ahead and update both of them.

Reply:
I just found it odd that we use 3.1 everywhere else, but 3.0 here...

Reply:
I have switched it to 3.1.